src/test/java/seedu/guest/model/guest/BillTest.java

```diff
@@ -54,7 +57,8 @@ public void isValidBill() {
         assertTrue(Bill.isValidBill("1.5")); // 1 decimal place
```

Comment:
Should we add decimal point with 1 decimal place? (eg. .9)

Reply:
Yes, added